types/chance/index.d.ts

```diff
@@ -225,13 +236,17 @@ declare namespace Chance {
         name: string;
         abbreviation: string;
     }
-}
 
-// window.chance
-declare var chance: Chance.Chance;
-declare var Chance: Chance.ChanceStatic;
```

Comment:
These two lines above assume everyone has chance installed globally. Deleting them as Typescript assumes this is good even when chance is not global